doc/how_to_guides/isaac_panda/isaac_panda_tutorial.rst

```diff
@@ -0,0 +1,169 @@
+How To Command Simulated Isaac Robot
+====================================
+
+This tutorial requires a machine with ``Isaac 2022.2.0`` installed.
+For Isaac requirements and installation please see the `Omniverse documentation <https://docs.omniverse.nvidia.com/app_isaacsim/app_isaacsim/overview.html>`_.
+
+This tutorial has the following assumptions on system configuration:
+
+1. Isacc 2022.2.0 is installed on a Ubuntu 20.04 host in the "$HOME/.local/share/ov/pkg/isaac_sim-2022.2.0" directory. (this is the default location)
+2. Docker is installed.
```

Comment:
If you don't get rid of the GPU stuff in the compose file for some reason, this also has to include the NVIDIA Container Toolkit https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html